AWSClientRuntime/Tests/Middlewares/Sha256TreeHashMiddlewareTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import ClientRuntime
+import AwsCommonRuntimeKit
+@testable import AWSClientRuntime
+
+class Sha256TreeHashMiddlewareTests: XCTestCase {
+    func testTreeHashAllZeroes() {
+        let context = HttpContextBuilder().build()
+        let byteArray: [UInt8] = Array(repeating: 0, count: 5767168)
+        let byteBuffer = ByteBuffer(bytes: byteArray) // 5.5MB buffer
```

Comment:
is there something special about a 5.5MB buffer?  if not, plz remove this comment.

Reply:
I see, we could have a variable called
```
let bytesIn5_5MB: Int = 1024*1024*5.5
```